src/recon_buildblock/distributable.cxx

```diff
@@ -185,10 +185,10 @@ void get_viewgrams(shared_ptr<RelatedViewgrams<float> >& y,
 #if !defined(_MSC_VER) || _MSC_VER>1300
       additive_binwise_correction_viewgrams.reset(
         new RelatedViewgrams<float>
-        (binwise_correction->get_related_viewgrams(view_segment_num, symmetries_ptr, false, timing_pos_num)));
+        (binwise_correction->get_related_viewgrams(view_segment_num, symmetries_ptr, false)));
 #else
       RelatedViewgrams<float> tmp(binwise_correction->
-                                  get_related_viewgrams(view_segment_num, symmetries_ptr, false, timing_pos_num));
+                                  get_related_viewgrams(view_segment_num, symmetries_ptr, false, view_segment_num.tof_pos_num()));
```

Comment:
probably not future proof, but I assume this code isn't used, we don't support MSC_VER 1300 anymore (VS 2003!). Probably better to remove the `'#ifdef` completely

Reply:
Removed the `#ifdef`